test/src/tests-persistence.ts

```diff
@@ -1,6 +1,6 @@
 import Utils from './config/utils';
 import sinon from 'sinon';
-import fetchMock from 'fetch-mock/esm/client';
+import fetchMock, { done } from 'fetch-mock/esm/client';
```

Comment:
seems like you are removing all the `done`s here in favor of `await`-ing.  Why is `done` being imported?